src/eutils.jl

```diff
@@ -104,7 +105,8 @@ Parameters: db, id, query_key, WebEnv, retstart, retmax, retmode, version.
 """
 function esummary(ctx::Associative=empty_context(); params...)
     params = process_parameters(params, ctx)
-    return Requests.post(string(baseURL, "esummary.fcgi"), data=params)
+    body = HTTP.escapeuri(params)
+    return HTTP.request("POST", string(baseURL, "esummary.fcgi?", body))
 end
 
```

Comment:
Not sure if it makes a difference, but should the body be passed as a parameters instead of as part of the url?